### PR TITLE
Rails Basics/Express & Mongoose Deployment Lesson: Update costs associated with Fly.io (they updated their pricing)

### LINKED ISSUE #27213: Rails Basics/Express & Mongoose: Update costs associated with Fly.io (they updated their pricing)
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

Fly.io recently announced that they will begin charging for stopped machines, based on the disk space it's using. 
https://community.fly.io/t/we-are-going-to-start-collecting-charges-for-stopped-machines-rootfs-starting-april-25th/17825

All the details are in that announcement, but it is to my understanding that your project will not cost $0. Since our usage would be very, very low, I imagine that the vast majority of users would pay close to $0 anyways, but it might be nice to mention this so no one is surprised when time comes for billing.

Both of these would need a slight update in wording:
The Rails deployment lesson: https://www.theodinproject.com/lessons/ruby-on-rails-deployment
The Node.js deployment lesson: https://www.theodinproject.com/lessons/nodejs-deployment

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/ruby-on-rails-deployment

### (Optional) Discord Name

Eduardo06sp

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by maahsnd**
## Because

Fly.io will be changing their pricing structure. Specifically, on April 25 they will begin charging $0.15/GB of RootFS for a machine stopped for 30 days. However, on the free plan invoices under $5 USD are waived, meaning that TOP students are very unlikely to be charged. This PR adds an explanation of the pricing change to the deployment lesson in both the Ruby on Rails and NodeJS courses, and will hopefully alleviate any student concerns in the event that they see an invoice.

## This PR

- Added a bullet point stating the charge rate for stopped machines to the list immediately beneath the Fly.io header for both the Ruby on Rails and NodeJS deployment lessons.
- Added a bullet point explaining that invoices under $5 USD are waived, and that students are thus very unlikely to be charged. This bullet point is added to the list beneath the Free plan header for both the Ruby on Rails and NodeJS deployment lessons.
- Corrected capitalization of JavaScript in both lesson files. The s was lowercase, and is now uppercase.

## Issue

Closes #27213 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: ruby_on_rails/rails_basics/deployment.md [hunk 1]
```diff
@@ -208,21 +225,20 @@ This will be where the Git skills you've been learning will start to really pay
 <div class="lesson-content__panel" markdown="1">
 
 1. Deploy your [Blog App project](https://www.theodinproject.com/lessons/ruby-on-rails-blog-app) to one of the hosting providers we've mentioned. If you need help deciding which one to use, we recommend Fly.io. The important thing to take away from this first deployment is getting experience deploying. Don't worry if you don't understand everything that's happening. That will come with time.
-    * Use one of the linked deploy guides for your PaaS provider to help you through the process.
-    * If you're having trouble deploying, check out the [Debugging and Troubleshooting Deployments](#debugging-and-troubleshooting-deployments) section for some tips.
+_ Use one of the linked deploy guides for your PaaS provider to help you through the process.
+_ If you're having trouble deploying, check out the [Debugging and Troubleshooting Deployments](#debugging-and-troubleshooting-deployments) section for some tips.
```
Comment (JoshDevHub) [inline ruby_on_rails/rails_basics/deployment.md:229]:
```suggestion
   - Use one of the linked deploy guides for your PaaS provider to help you through the process.
   - If you're having trouble deploying, check out the [Debugging and Troubleshooting Deployments](#debugging-and-troubleshooting-deployments) section for some tips.
```
These are meant to be bullet points under the numbered item just above.

**Review by JoshDevHub — CHANGES_REQUESTED** [review]
Awesome work :raised_hands:

Thank you for taking the time to go ahead and fix the linting issues that appear in these lessons. Just going to recommend one small fix with one of the lint fixes and then we'll be good to merge :rocket:

**Comment by JoshDevHub** [comment]
Also we'll not worry about the "Emphasis as heading" issues in this PR, as that might require some extra thought to clean up. So just that one fix I pointed out above and I'll merge.

**Comment by maahsnd** [comment]
Thanks Josh!  I committed your requested change. Anything else I need to do here?

**Review by JoshDevHub — APPROVED** [review]
@maahsnd Looks great. Thank you for your contribution :raised_hands:

**Pull request merged**